docs/csharp/programming-guide/classes-and-structs/named-and-optional-arguments.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Named and Optional Arguments - C# Programming Guide"
+description: Named arguments in C# specify arguments by name, not position. Optional arguments can be omitted. Named & optional arguments facilitate calls to COM interfaces.
```

Comment:
Let's drop the last sentence. Named and optional arguments will be much more useful for records (coming in C# 9)
```suggestion
description: Named arguments in C# specify arguments by name, not position. Optional arguments can be omitted.
```